fintorch/datasets/ellipticpp.py

```
@@ -395,11 +399,11 @@ def __init__(
         num_test: float = 0.1,
         disjoint_train_ratio: float = 0.3,
         neg_sampling_ratio: float = 2.0,
-        num_neighbors: List[int] = None,
+        num_neighbors: List[int] = [10, 30],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)